frontend/src/__tests__/components/TopBar/PopUps/AccessibilityDialog.test.tsx

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
+import {describe, test} from 'vitest';
 import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import i18n from '../../../../util/i18nForTests';
-
 import {I18nextProvider} from 'react-i18next';
 import ApplicationMenu from '../../../../components/TopBar/ApplicationMenu';
 
 describe('AccessibilityDialog', () => {
-  test('PopUp', () => {
-    render(
-      <I18nextProvider i18n={i18n}>
-        <ApplicationMenu />
-      </I18nextProvider>
-    );
-    screen.getByLabelText('topBar.menu.label').click();
-    screen.getByText('topBar.menu.accessibility').click();
+  test('PopUp', async () => {
+    render(<I18nextProvider i18n={i18n}>{<ApplicationMenu />}</I18nextProvider>);
```

Comment:
Not sure why there are {} around this.
```suggestion
    render(
      <I18nextProvider i18n={i18n}>
        <ApplicationMenu />
      </I18nextProvider>
    );
```